include/hcl/concurrent/unordered_map/memory_allocation.h

```diff
@@ -0,0 +1,130 @@
+#ifndef __MEMORY_ALL_H_
```

Comment:
What is this memory allocators?

Reply:
They are dynamic memory allocators with reuse. They are used by the unordered map and skiplist implementations.